diff --git a/cpp/CMakeLists.txt b/cpp/CMakeLists.txt
index 311de52..b9c4506 100644
--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -9,6 +9,8 @@ set(CMAKE_CXX_STANDARD 11)
 
 set(CMAKE_C_STANDARD 99)
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
 # Download and unpack googletest at configure time
 configure_file(CMakeLists.txt.in googletest-download/CMakeLists.txt)
 execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
